NEWS.md

```diff
@@ -37,27 +43,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * Improved distribution of random vector generation for rotation matrices and complex circle.
 
-## [0.9.7] - 2023-11-14
+## [0.9.7] * 2023-11-14
```

Comment:
Why did you change `-` to `*`?

Reply:
oh that was by accident. I changed itemise points to a unified format, I think/hope. Will check that.